src/cmdline.c

```diff
@@ -96,7 +96,7 @@ int cmdline_run(const char **argv, int argc)
 	 * a null pointer in the array. Because of this, AppKit in OS X 10.10 will
 	 * dereference it, causing a segmentation fault.
 	 */
-	char** mutableArgv = malloc(argvsize);
+	const char** mutableArgv = malloc(argvsize);
```

Comment:
Because of pointer-to-pointer, and only the the pointee is const, hence the first-level pointer is a "pointer to pointer to const char", while in the other case it is "pointer to pointer to char", hence incompatibilty